metricbeat/scripts/mage/target/metricset/metricset.go

```diff
@@ -0,0 +1,58 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package metricset
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/magefile/mage/sh"
+
+	devtools "github.com/elastic/beats/v7/dev-tools/mage"
+)
+
+// CreateMetricset creates a new metricset.
+//
+// Required ENV variables:
+// * MODULE: Name of the module
+// * METRICSET: Name of the metricset
+func CreateMetricset() error {
```

Comment:
For consistency with other targets' functions, how about taking a `context.Context` here as well?

Reply:
Being that it's not using it, I removed it. Should we have it there even if its not being used?

Reply:
Yeah, I could go either way over here. I see that other targets have it and it's not being used in them as well. Personally, I'd prefer to leave it out if it's not being used but perhaps we can do that everywhere in a follow up PR.